release_tag.toml

```diff
@@ -0,0 +1,24 @@
+# disables labeling when this label is on a PR
+disable_on = "no_release"
+
+# The priority is used when only_highest_priority is set to true
+labels = [
+  { name = "patch_release", priority = 1 },
+  { name = "minor_release", priority = 2 },
+  { name = "major_release", priority = 3 },
+]
+
+# group within groups must be defined in `changelog.toml`
+# label within groups must be defined in `labels`
+groups = [
+  { group = "Added", label = "minor_release" },
+  { group = "Changed", label = "major_release" }, # maybe minor?
+  { group = "Removed", label = "major_release" }, # maybe minor?
+  { group = "Bug Fixes", label = "patch_release" },
+]
```

Comment:
I think Changed and Removed should always be major releases and only be used if you change something API (e.g. rename or remove a field) or system related (e.g. deprecation of ospd-openvas)

Reply:
Changed or Removed should only be used when it is doing that. Keep in mind those will be rendered in the changelog and is confusing otherwise. I also used it wrongly before.